assets/src/dashboard/app/views/savedTemplates/content/index.js

```diff
@@ -58,27 +58,25 @@ function Content({
     'enableInProgressStoryActions'
   );
 
-  const handleMenuItemSelected = useCallback(
-    (sender, template) => {
-      setContextMenuId(-1);
-
-      switch (sender.value) {
-        case SAVED_TEMPLATE_CONTEXT_MENU_ACTIONS.OPEN_IN_EDITOR:
-          trackEvent('use_saved_template', {
-            name: template.title,
-            template_id: template.id,
-          });
-          actions.createStoryFromTemplate(template);
-          break;
-
-        case SAVED_TEMPLATE_CONTEXT_MENU_ACTIONS.PREVIEW:
-          actions.previewTemplate(null, template);
-          break;
+  // TODO add context menu actions
```

Comment:
note can be removed, i think you created these?

Reply:
The diff is weird, but this was here before. If you scroll down a little you'll see:
![image](https://user-images.githubusercontent.com/22185279/109321931-f2019f80-780e-11eb-9875-4b16e79b04c4.png)

But we don't need this comment anymore, the context menu has all of its actions